common/changes/@itwin/viewer-react/AddCornerButtonBackstage_2023-08-18-17-10.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "Expose corner button for defaultUiConfig and add backstage corner button when there are backstage items provided",
+      "type": "patch"
```

```suggestion
      "type": "minor"
```

while although this started off as a bugfix, since you are modifying the api, i would make this a minor